composer.json

```diff
@@ -21,7 +21,7 @@
         "doctrine/orm": "~2.4",
         "doctrine/doctrine-bundle": "~1.2",
         "doctrine/doctrine-fixtures-bundle": "~2.2",
-        "fabpot/php-cs-fixer": "~2.0@dev",
+        "friendsofphp/php-cs-fixer": "~1.0",
```

Comment:
downgrading it is not good, there is configuration changes between 1.x and 2.x, keep it at `~2.0@dev`